sys/net/gnrc/pktbuf/Makefile.include

```diff
@@ -0,0 +1,4 @@
+# Check that only one implementation of pktbuf is used
+ifneq (1,$(words $(filter gnrc_pktbuf_malloc gnrc_pktbuf_static,$(USEMODULE))))
```

Comment:
```suggestion
ifneq (1,$(words $(filter gnrc_pktbuf_%,$(USEMODULE))))
```
?

This way this file does not need adaptation when there are more `gnrc_pktbuf` implementations (e.g. the long awaited `memarray` version or a `gnrc_pktbuf_mock` I had in my mind)

Reply:
gnrc_pktbuf_cmd is also a thing

Reply:
The command module could be renamed. But besides that I usually think that being explicit tends to be clearer on the long run. Do you expect this list to grow a lot?

Reply:
Not a lot, but I expect it to grow :-). I think for feature that is about warning the user, we should be rather as broad as possible, so maybe add a filter option to exclude `gnrc_pktbuf_cmd` here?

Reply:
Ok, I've added a filter for the command module, and now implementations are caught using a wildcard.